lib/yara/scan_result.rb

```diff
@@ -0,0 +1,75 @@
+module Yara
+  class ScanResult
+    RULE_MATCHING     = 1
+    RULE_NOT_MATCHING = 2
+    SCAN_FINISHED     = 3
```

Comment:
I kinda wish we didn't create a ScanResult for the SCAN_FINISHED callback_type. What do you think about only constructing one if RULE_MATCHING or RULE_NOT_MATCHING?

Reply:
I like that 👍 Updated in https://github.com/jonmagic/yara-ffi/pull/4/commits/90cd1acacfcab1f268012c1f6b71d1fa7d25b860